### ISSUE TITLE
minerCategory Conditionally Checked using Non-strict comparison with zero

### ISSUE THREAD

**Issue by bearcanrun** [Superalgos/ALGOToken#20]
Variables of uint type cannot be negative. Thus, comparing uint variable with zero (greater than or equal) is redundant. In AlgoPool and AlgoFees, minerCategory is conditionally checked in a `require()` statement using `>= 0`.

This issue was flagged by [SmartCheck](https://tool.smartdec.net/scan/e3c7fbd47ddb46478e9ec131a222a519), but would only be a real issue if was used as condition in a for-loop causing infinite loop/underflow issues.

**Code:** 
```
# AlgoPool.sol(L:38), AlgoFees.sol(L:42)

require(minerCategory >= 0 && minerCategory <= 5);

// Recommended
require(minerCategory = 0 || minerCategory <= 5);

```

**Comment by pmmax** [comment]
Fixed in #21

**Comment by bearcanrun** [comment]
Documentation updated.
